anyway/flask_app.py

```diff
@@ -2050,3 +1418,90 @@ def add_user_to_org() -> Response:
 #     return Response(json_data, mimetype="application/json")
 #
 #
+
+# User system API
+@login_manager.user_loader
```

Comment:
I do not know what are these annotations `@login_manager`, `@identity_loaded`, etc. If they are not HTTP endpoint URLs, maybe it is an opportunity to move them out of `flask_app.py`?

Reply:
I will move those functions out of `flask_app.py`